Packs/AnsibleTower/Playbooks/Ansible_Tower-_ad_hoc_command-_generic_polling.yml

```diff
@@ -302,7 +307,7 @@ inputs:
   playbookInputQuery:
 - key: Timeout
   value:
-    simple: "15"
+    simple: "10"
```

Comment:
@idovandijk Why did you change the timeout from 15 to 10 as the default value?

Reply:
The playbook input was set to 15 but the task was hardcoded to use 10, so in order to preserve BC, I changed the input to be the same value that previously was hardcoded (10)